plugins/kubernetes/db/migrate/20150827181056_create_kubernetes_cluster.rb

```diff
@@ -0,0 +1,11 @@
+class CreateKubernetesCluster < ActiveRecord::Migration
+  def change
+    create_table :kubernetes_clusters do |t|
+      t.string :name
+      t.string :description
+      t.string :config_filepath
```

Comment:
This is a local filepath? Are you expecting this to be written out by a config management system? Can Samson be the source of truth here?